File: src/test/java/pt/whd/readFile/AppTest.java
package pt.whd.readFile;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;

import org.junit.Test;

import java.io.IOException;

/**
 * Unit test for simple App.
 */
public class AppTest 
{
    /**
     * Rigorous Test :-)
     */
    @Test
    public void isReadingFileProperly() throws IOException {
        String ids = App.readFile("arquivo_teste.txt");

        assertEquals( ids, "38703,39134,39287");
    }
}

File: src/test/java/pt/whd/readFile/AppTest.kt
package pt.whd.readFile

import org.junit.Assert
import org.junit.Test

/**
 * Unit test for simple App.
 */
class AppTest {


    @Test
    fun isReadingFileProperly() {
        val ids = readFile("arquivo_teste.txt")
        Assert.assertEquals(ids, "38703,39134,39287")

    }
}